test/integration/kubernetes.test.ts

```diff
@@ -143,7 +143,7 @@ tap.test('snyk-monitor sends binary hashes to kubernetes-upstream after adding a
   t.equals(nodePluginResult.hashes.length, 1, 'one key-binary hash found in node image');
   t.equals(
     nodePluginResult.hashes[0],
-    '1fd7117ccedc1c673381f71cc611131cb8a47852a882e3d731381f14355a3346',
+    '45cf3343b95cb067dec6d64ccc848f48b598baffed5b12be1d14142d087da0e8',
     'SHA256 for node v12.16.1 found',
```

Comment:
is it the same node version as well? (12.16.1)

Reply:
another option is to just remove the version from here, not sure it is interesting

Reply:
no, good spot, it's ```v12.16.2```